File: app_update/src/main/java/com/example/app_update/update/net/IDownloadCallback.java
package com.example.app_update.update.net;

import java.io.File;

/**
 * 下载管理接口类。
 */
public interface IDownloadCallback {
    void onSuccess(File apkFile);

    void onFailed(Throwable throwable);

    void onProcess(int process);
}

File: app_update/src/main/java/com/example/app_update/update/net/IDownloadCallback.kt
package com.example.app_update.update.net

import java.io.File

/**
 * 下载管理接口类。
 */
interface IDownloadCallback {
    fun onSuccess(apkFile: File?)
    fun onFailed(throwable: Throwable?)
    fun onProcess(process: Int)
}